0it [00:00, ?it/s]
0it [00:00, ?it/s]

Traceback (most recent call last):
  File "train.py", line 160, in <module>
    pieces = TrainerPieces.from_params(params, serialization_dir, args.recover)  # pylint: disable=no-member
  File "/local/tprakap/am-parser/graph_dependency_parser/train/amtrainer.py", line 825, in from_params
    all_datasets = training_util.datasets_from_params(params)
  File "/home/tprakap/.conda/envs/docker_allennlp/lib/python3.7/site-packages/allennlp/training/util.py", line 178, in datasets_from_params
    train_data = dataset_reader.read(train_data_path)
  File "/home/tprakap/.conda/envs/docker_allennlp/lib/python3.7/site-packages/allennlp/data/dataset_readers/dataset_reader.py", line 134, in read
    instances = [instance for instance in Tqdm.tqdm(instances)]
  File "/home/tprakap/.conda/envs/docker_allennlp/lib/python3.7/site-packages/allennlp/data/dataset_readers/dataset_reader.py", line 134, in <listcomp>
    instances = [instance for instance in Tqdm.tqdm(instances)]
  File "/home/tprakap/.conda/envs/docker_allennlp/lib/python3.7/site-packages/tqdm/std.py", line 1195, in __iter__
    for obj in iterable:
  File "/local/tprakap/am-parser/graph_dependency_parser/components/dataset_readers/amconll_automata.py", line 103, in _read
    for instance in self._read_one_file(formalism, path):
  File "/local/tprakap/am-parser/graph_dependency_parser/components/dataset_readers/amconll_automata.py", line 90, in _read_one_file
    yield self._read_one_sentence(formalism, automata_zip_reader, i, am_sentence)
  File "/local/tprakap/am-parser/graph_dependency_parser/components/dataset_readers/amconll_automata.py", line 93, in _read_one_sentence
    automaton = automata_zip_reader.readAutomaton(position_in_corpus)
  File "jnius/jnius_export_class.pxi", line 769, in jnius.JavaMethod.__call__
  File "jnius/jnius_export_class.pxi", line 846, in jnius.JavaMethod.call_method
  File "jnius/jnius_utils.pxi", line 91, in jnius.check_exception
jnius.JavaException: JVM exception occurred: entry
